.github/workflows/release-postman.yml

```diff
@@ -1,39 +1,28 @@
 name: Postman Release
 on:
-  schedule:
-    - cron: '0 10 ? * *'
+  pull_request:
+    branches: 
+      - main
+    types:
+      - closed
   workflow_dispatch:
 permissions:
   issues: write
 
 jobs:
   release-postman:
+    if: ${{ github.head_ref == 'api-bot-update' && github.event.pull_request.merged }}
```

Comment:
ah okay, here you are checking for PRs coming from the user used for the release

Reply:
Yeah it will run the workflow on every merge to master but the main job itself will only run on release PR merges. This was the only way I could find to do it on the [github docs](https://www.mongodb.com/developer/products/atlas/admin-api-postman/)